app/controllers/admin/whitelists_controller.rb

```diff
@@ -21,6 +21,15 @@ def create
         presenter: UseCases::Administrator::CreateSignupWhitelist.new
       ).execute
 
+      UseCases::PublishOrganisationWhitelist.new(
```

Comment:
This is in the wrong controller. Its a list of organisation names, not a whitelist